Tribler/Core/Modules/restapi/multichain_endpoint.py

```diff
@@ -309,8 +312,7 @@ def render_GET(self, request):
 
         mc_community = self.get_multi_chain_community()
         nodes, edges = mc_community.get_graph(focus_node, neighbor_level)
-
-        return json.dumps({"focus_node": focus_node,
+        return json.dumps({"focus_node": hexlify(focus_node),
```

Comment:
Instead of changing all the "hexlified" focus_node's into unhexlified form, why not only do this in line 314 i.e. where mc_community is called?